mycroft/skills/common_iot_skill.py

```diff
@@ -155,30 +191,40 @@ def __init__(self,
         self.attribute = attribute
         self.entity = entity
         self.scene = scene
+        self.value = value
 
     def __repr__(self):
         template = ('IoTRequest('
                     'action={action},'
                     ' thing={thing},'
                     ' attribute={attribute},'
                     ' entity={entity},'
-                    ' scene={scene}'
+                    ' scene={scene},'
+                    ' value={value}'
                     ')')
         return template.format(
             action=self.action,
             thing=self.thing,
             attribute=self.attribute,
             entity='"{}"'.format(self.entity) if self.entity else None,
-            scene='"{}"'.format(self.scene) if self.scene else None
+            scene='"{}"'.format(self.scene) if self.scene else None,
+            value='"{}"'.format(self.value) if self.value else None
```

Comment:
Just realized another thing... if value is 0 this will fail and set the value to None. Or is that intended?